lib/routes/xueqiu/timeline.ts

```diff
@@ -0,0 +1,88 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import { config } from '@/config';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+const rootUrl = 'https://xueqiu.com';
+export const route: Route = {
+    path: '/timeline',
+    categories: ['finance'],
+    example: '/xueqiu/timeline',
+    parameters: {},
+    features: {
+        requireConfig: [
+            {
+                name: 'XUEQIU_COOKIES',
+                description: '',
+            },
+        ],
+        requirePuppeteer: false,
+        antiCrawler: true,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '用户关注时间线',
+    maintainers: ['ErnestDong'],
+    handler,
+    description: `:::warning
+  用户关注动态需要登录后的 Cookie 值，所以只能自建，详情见部署页面的配置模块。
+  :::`,
```

Comment:
Please do mention what `usergroup_id` means in the docs

Reply:
Changed `usergroup_id` to `param` instead of `query` and added docs. I'll push to demo later.